canonical-websites/www.ubuntu.com
# Add resources strip to /openstack

## Done

Add resources strip to /openstack

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack](http://0.0.0.0:8001/openstack)
- Check that the new 'resources' section has been added and is as per copy doc

## Issue / Card

Fixes: #2407 
Doc: https://docs.google.com/document/d/12z6QEGpqMVbNPJOd5aUfn0VnXQ0Sa5YriEBLA7VGNwI/edit#
